src/test/java/core/basesyntax/impl/WriterServiceImplTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.impl;
+
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class WriterServiceImplTest {
+    private static final String REPORT_FILE_PATH = "src/main/resources/report.csv";
+    private WriterServiceImpl writerService;
+    private ReaderServiceImpl readerService;
```

Comment:
Consider utilizing Java's built-in file reading utilities to minimize dependencies and enhance test independence